Source/Libraries/GSF.Core/Data/DataInserter.cs

```diff
@@ -437,7 +434,8 @@ private bool ClearTable(Table table)
                     return ClearTable(table);
                 }
 
-                OnSQLFailure(deleteSql, ex);
+                if (useTruncateTable) OnSQLFailure("TRUNCATE TABLE " + table.SQLEscapedName, ex);
+                else OnSQLFailure("DELETE FROM " + table.SQLEscapedName, ex);
```

Comment:
Parameter substitution does not work on Tables

Reply:
updated